index.html

```diff
@@ -2388,9 +2389,13 @@ <h2 id="AudioParam">
               "#widl-AudioParam-exponentialRampToValueAtTime-void-float-value-double-endTime">
               exponentialRampToValueAtTime</a></code>, respectively.
             </p>
+            <p>
+              This method returns the current <a>AudioParam</a> object for easy
```

Comment:
No, I think you need to say what the actual returned object is.  I would just remove the easy chaining part.